Report/Chapters/Recommendations.tex

```diff
@@ -1 +1,18 @@
 \chapter{Recommendations}
+
+Wile working on this project we came up with a lot of ideas.
+Sadly, a large number of these ideas were too big or too complicated to be included in this project.
+However, future developers might find these ideas useful as inspiration.
+
+We have compiled a wish list of the features we would like to see in Viskell:
+
+\begin{itemize}
+	\item Circular menu (see \ref{circular_menu} on page \ref{circular_menu}).
+	\item Compact view of functions (for large programs).
+	\item Support for more Haskell functions (e.g. I/O).
+	\item Support for recursion and pattern matching.
```

Comment:
Nitpick: what's missing is *explicit* recursion. We have implicit recursion, i.e. through functions like `until`.